link to version that exports module

blairanderson opened the pull request:
Hi There, 

I wanted this but without adding to String.

Wasn't sure if people asked you for that much or not...  ¯\_(ツ)_/¯

bryanwoods commented:
The two downsides I see to monkeypatching `String` are:

1) A potential name collision with a preexisting `autoLink` method, and
2) The code is executing in a JavaScript environment that does not have a `String` prototype for some reason

The first downside seems completely possible but a reasonable tradeoff (you're already OK with monkey patching core prototypes and you just have to decide which `autoLink` function you prefer), and the second seems more unlikely.

Is there another case I'm not considering? I know a lot of developers get uncomfortable with this type of extension, but making a breaking change to the API for that reason doesn't seem compelling enough to me at the moment.

Let me know what you're thinking and thanks for opening this issue.

blairanderson commented:
Yeah I didn't think it warranted a breaking change so I forked and converted and added a link to it.

There are a bunch of people that don't like monkey patching, I'm one that tries to stay away from it.
